content/docs/2.7/scalers/azure-data-explorer.md

```diff
@@ -0,0 +1,214 @@
++++
+title = "Azure Data Explorer"
+layout = "scaler"
+availability = "v2.7+"
+maintainer = "Community"
+description = "Scale applications based on Azure Data Explorer query result."
+go_file = "azure_data_explorer_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `azure-data-explorer` trigger that scales based on an Azure Data Explorer query result.
+
+```yaml
+triggers:
+- type: azure-data-explorer
+  metadata:
+    podIdentity: "azure" # Provide either podIdentity or aad app creds. can use TriggerAuthentication as well
+    clientId: "AAD_APP_CLIENT_ID" # Provide either podIdentity or aad app creds. can use TriggerAuthentication as well
+    clientSecret: "AAD_APP_SECRET" # Provide either podIdentity or aad app creds. can use TriggerAuthentication as well
+    tenantId: "AAD_APP_TENANT_ID" # Provide either podIdentity or aad app creds. can use TriggerAuthentication as well
+    databaseName: "DATA_EXPLORER_DATABASE_NAME"
+    endpoint: "https://<ClusterName>.<Region>.kusto.windows.net"
```

Comment:
Does this mean we only support public cloud?

Reply:
Nope, I changed it